ImagePickerSheetController/ImagePickerSheetController/ImagePickerSheetController.swift

```diff
@@ -31,13 +31,19 @@ public enum ImagePickerMediaType {
     
 }
 
+fileprivate let imageManager = PHCachingImageManager()
```

Comment:
The underlying issue turns out to be related to a well documented behavior in PHCachingImageManager - http://stackoverflow.com/questions/33266346/photos-framework-crash-this-application-is-not-allowed-to-access-photo-data. Nice find @colinta :metal:.

This should prevent the crash for all users. We plan to go a step further and guard against displaying the picker when the user has denied access to the photos library. We'll display a message to the user directing them to make the change in their settings.